.changeset/green-donkeys-change.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/draft-modal": patch
+---
+
+Fix: A rendering bug when labels for actions were the same (even for short period of time)
```

Comment:
Seems like a flag if you're giving two buttons the same label, what's the exact scenario you're trying to fix?

Reply:
It's there just for split second and breaks react.
`i18n-react-intl` shows during locale loading all strings as `" "` which means they are the same. When the right labels are loaded the components get rerendered. But by then it's already broken.